modules/kvformat/kv-parser.c

```diff
@@ -99,8 +102,10 @@ _process(LogParser *s, LogMessage **pmsg, const LogPathOptions *path_options, co
 {
   KVParser *self = (KVParser *) s;
   KVScanner kv_scanner;
+  self->init_scanner(self, &kv_scanner);
+  if (self->prefix)
+    g_string_assign(kv_scanner.formatted_key, self->prefix);
 
-  kv_parser_init_scanner(self, &kv_scanner);
   log_msg_make_writable(pmsg, path_options);
```

Comment:
It is a convention that we have an inline function around virtual methods, why do we get rid of this?